src/libs/Navigation/AppNavigator/index.js

```diff
@@ -1,12 +1,27 @@
 import PropTypes from 'prop-types';
-import React from 'react';
+import React, {useContext, useEffect} from 'react';
+import {NativeModules} from 'react-native';
+import InitialUrlContext from '@libs/InitialUrlContext';
+import Navigation from '@libs/Navigation/Navigation';
 
 const propTypes = {
     /** If we have an authToken this is true */
     authenticated: PropTypes.bool.isRequired,
 };
 
 function AppNavigator(props) {
+    const initUrl = useContext(InitialUrlContext);
+
+    useEffect(() => {
+        if (!NativeModules.ReactNativeModule) {
+            return;
+        }
+
+        Navigation.isNavigationReady().then(() => {
+            Navigation.navigate(initUrl);
```

Comment:
Is there a difference between `null` and `undefined` here? Or was there no particular reason that you used `null`?

Reply:
What do you mean by that? I used `null` as initial value for the context that contains `initUrl`, but this part of code shouldn't be executed anyway if no url is passed in `initialProps`

Reply:
Yeah, I'm asking if there is a reason why is `null` the initial value for the context, not `undefined`.

Reply:
Actually I have just changed it to `undefined`, because after giving it more thought it is better then `null` (if no url is passed in initialProps, it's `undefined` anyway